### PR TITLE
Flushes page cache sequentially during some import stages

### PR DESCRIPTION
**Pull request by tinwelint**
To get maximum sequential write I/O. Node and Relationship stages gets
a throughput improvement up to 2x or more, depending on hardware.

### CODE REVIEW

File: community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java [hunk 1]
```diff
@@ -303,8 +304,8 @@ private void importRelationships( NodeRelationshipCache nodeRelationshipCache,
 
         long nextRelationshipId = 0;
         Configuration relationshipConfig = withBatchSize( config,
-                neoStore.getRelationshipStore().getRecordsPerPage() );
-        Configuration nodeConfig = withBatchSize( config, neoStore.getNodeStore().getRecordsPerPage() );
+                neoStore.getRelationshipStore().getRecordsPerPage() * 100 );
+        Configuration nodeConfig = withBatchSize( config, neoStore.getNodeStore().getRecordsPerPage() * 100 );
```
Comment (chrisvest) [inline community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java]: A wild `* 100` appears, with no apparent explanation.

Comment (tinwelint) [reply community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java]: Didn't think that got it in. I'll split that out to another commit and clean that up

**Comment by tinwelint** [comment]
The second commit went in here as to reduce number of PRs coming for the importer. It's fairly small.

**Review by chrisvest — APPROVED** [review]
Looks good to me! 👍

**Pull request closed without merging**